Design/Workbench/Plot-Selector/design-plot-selector.md

```diff
@@ -0,0 +1,73 @@
+# Plot Selector Widget
+
+## Requirements
+
+The request from the January 2018 user meeting was to have a widget that allowed easy switching between plots, in particular for SANS and Reflectometry users. Another request from the ILL was to have a method to easily arrange the plots.
+
+For consistency the design should follow as closely as possible similar functionality in Mantid, such as the workspace selector and plotting options within the workspace selector.
+
+## Functional Design
+
+### Interface Design
+
+For consistency the plot selector widget should follow a similar design to the workspace selector. Double-clicking a plot name will bring the plot to the foreground and make it the active one.
+
+![Plot Selector Widget](plot_selector.png)
+
+**All plots created from the workbench are shown in the list.** The name shown will be the figure window title. Double-clicking a plot will bring it to the foreground. Filtering works as for the workspace selector. Close can act on one or more selected plots. Additionally plots can be closed individually by clicking the close button on one of the plots in the list.
```

Comment:
Do we create unique names? If yes, is it easy for the user to select the correct plot just from the name? For example, I can have plot1 containing workspace w, spectrum 1, plot 2 can have color fill plot of the same workspace, and plot 3 also spectra 1 of the same workspace. If the figure names are `w-1`, `w-2`, `w-3`, selecting plots by name would be kind of difficult.

Reply:
A good question, I have been assuming we do have unique names. In MantidPlot the current behaviour is to do `[ws_name]-1`, `[ws_name]-2` etc. The names seem to be unique. In the workbench the figures are currently `Figure 1`, `Figure 2` etc. Perhaps the automatic naming should also include the plot type, or spectrum number?